eva_submission/eload_utils.py

```diff
@@ -162,3 +162,16 @@ def get_hold_date_from_ena(project_accession, project_alias=None):
         if not hold_date:
             raise ValueError(f"Couldn't get hold date from ENA for {project_accession} ({project_alias})")
     return hold_date
+
+
+def backup_file(file_name):
+    suffix = 1
+    backup_name = file_name + '.' + str(suffix)
+    while os.path.exists(backup_name):
+        suffix += 1
+        backup_name = file_name + '.' + str(suffix)
+
+    for i in range(suffix, 1, -1):
```

Comment:
```suggestion
    for i in range(suffix, 2, -1):
```
I _think_ this is what you want, since there's never a `{filename}.0`.

Reply:
`range` is end exclusive which mean for a suffix of 5 i would iterate `[5, 4, 3, 2]`

Reply:
I should know better than to make PR comments about indexing...